test/blackbox-tests/test-cases/describe_location.t

```diff
@@ -25,11 +25,13 @@ An executable from the current project:
   _build/default/foo.exe
 
 Test that executables from dependencies are located correctly:
-  $ mkdir dune.lock
-  $ cat > dune.lock/lock.dune << EOF
+
+  $ default_lock_dir="dune.lock"
```

Comment:
This is declared in `helpers.sh`. Ideally, it can be used directly from there, but if that's too complicated, it's okay to declare it again here. If this value changes in the future, the test will fail anyway to remind you to update it.